src/filters/dates/relative/relativeDates.test.ts

```diff
@@ -0,0 +1,216 @@
+            ],
+        }, {
+            description: 'should parse in a wk correctly',
+            case: 'go to work in a wk',
+            result: [
+                { dateTime: mockDates(currentDay + 7, currentMonth, currentYear), text: 'go to work', matched: 'in a wk' },
+            ],
+        }, {
+            description: 'should parse in X wks correctly',
+            case: 'go to work in 5 wks',
+            result: [
+                { dateTime: mockDates(currentDay + 35, currentMonth, currentYear), text: 'go to work', matched: 'in 5 wks' },
+            ],
+        }, {
+            description: 'should parse in a month correctly',
+            case: 'go to work in a month',
+            result: [
+                { dateTime: mockDates(currentDay, currentMonth + 1, currentYear), text: 'go to work', matched: 'in a month' },
+            ],
+        }, {
+            description: 'should parse in X months correctly',
+            case: 'go to work in 5 months',
+            result: [
+                { dateTime: mockDates(currentDay, currentMonth + 5, currentYear), text: 'go to work', matched: 'in 5 months' },
+            ],
+        }, {
+            description: 'should roll over to next year correctly when the year threshold is crossed',
+            case: 'go to work in 12 months',
+            result: [
+                { dateTime: mockDates(currentDay, currentMonth, currentYear + 1), text: 'go to work', matched: 'in 12 months' },
+            ],
+        }, {
+            description: 'should parse next month correctly',
+            case: 'go to work next month',
+            result: [
+                { dateTime: mockNext('month'), text: 'go to work', matched: 'next month' },
+            ],
+        }, {
+            description: 'should parse in a year correctly',
+            case: 'go to work in a year',
+            result: [
+                { dateTime: mockDates(currentDay, currentMonth, currentYear + 1), text: 'go to work', matched: 'in a year' },
+            ],
+        }, {
+            description: 'should parse in X years correctly',
+            case: 'go to work in 5 years',
+            result: [
+                { dateTime: mockDates(currentDay, currentMonth, currentYear + 5), text: 'go to work', matched: 'in 5 years' },
+            ],
+        }, {
+            description: 'should parse next year correctly',
+            case: 'go to work next year',
+            result: [
+                { dateTime: mockNext('year'), text: 'go to work', matched: 'next year' },
+            ],
+        }, {
+            description: 'should parse in a yr correctly',
+            case: 'go to work in a yr',
+            result: [
+                { dateTime: mockDates(currentDay, currentMonth, currentYear + 1), text: 'go to work', matched: 'in a yr' },
+            ],
+        }, {
+            description: 'should parse in X yrs correctly',
+            case: 'go to work in 5 yrs',
```

Comment:
You are going to say scope creep again, what about `go to work in five yrs`?

Reply:
Are you going to write a regex that will accept any verbal figure? That could be difficult.

Reply:
It is weird to allow "a" and "1" but not "one", don't worry about it for now I'll add an issue for this